advisories/2024-09-25_cometbft-bls12381-key-validation.md

```diff
@@ -0,0 +1,128 @@
+# Missing BLS12-381 Public Key Validation
+
```

Comment:
Let's remove the Impact section and rather have an introduction that summarises the issue, and have the Impact section as the second paragraph.